File: 240102/배수의 개수/number-of-multipliers.c
#include <stdio.h>

int main() {
    // 변수 선언
    int a, cnt3 = 0, cnt5 = 0;

	// 입력
	for(int i = 1; i <= 10; i++) {
		scanf("%d", &a);
		
		if(a % 3 == 0)
			cnt3++;
		if(a % 5 == 0)
			cnt5++;
	}

    // 출력
	printf("%d %d", cnt3, cnt5);

    return 0;
}

File: 241106/배수의 개수/number-of-multipliers.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        // 변수 선언
        int a, cnt3 = 0, cnt5 = 0;

        // 입력
        for(int i = 1; i <= 10; i++) {
            a = sc.nextInt();

            if(a % 3 == 0)
                cnt3++;
            if(a % 5 == 0)
                cnt5++;
        }

        // 출력
        System.out.print(cnt3 + " " + cnt5);
    }
}